pkg/ipamd/introspect.go

```diff
@@ -197,3 +197,20 @@ func getEnvBoolWithDefault(envName string, def bool) bool {
 	}
 	return def
 }
+
+func getEnvDurationWithDefault(envName string, def time.Duration) time.Duration {
+	inputStr, found := os.LookupEnv(envName)
+
+	if !found {
+		return def
+	}
+
+	if input, err := time.ParseDuration(inputStr); err == nil {
+		if input < 0 {
```

Comment:
I think we should cap the min to be `defaultNodeIPPoolReconcileInterval` for `NodeIPPoolReconcileInterval` and `defaultDecreaseIPPoolInterval` for `DecreaseIPPoolInterval`. Anything below the defaults will be way too aggressive and lot of ec2 calls.